900.version-fixes/p.yaml

```diff
@@ -14,6 +14,7 @@
 - { name: pangox-compat,               verlonger: 3,                 ruleset: rosa,        incorrect: true }
 - { name: paper-icon-theme,            verpat: "20[0-9]{2}.*",                             snapshot: true }
 - { name: paper-icon-theme,            verpat: "2017.*",                                   outdated: true } # 1.5.0 from 2018
+- { name: papirus-libreoffice-theme,   ver: "20170914",                                    devel: true } # that version was never tagged
```

Comment:
devel → incorrect + additional untrusted rule for aosc, please

Reply:
See fa31325